internal/constant/const.go

```diff
@@ -0,0 +1,4 @@
+package constant
```

Comment:
Should we call this errors, so that only and all error descriptions go in here?

Reply:
done